es-app/src/components/TextListComponent.h

```diff
@@ -37,14 +37,14 @@ class TextListComponent : public IList<TextListData, T>
 	using IList<TextListData, T>::stopScrolling;
 
 	TextListComponent(Window* window);
-	
+
```

Comment:
whitespace fixes / changes should be done in separate commit.

Reply:
Oops, my bad! I'll remove them!